app/controllers/GitController.scala

```diff
@@ -52,29 +37,23 @@ import play.api.libs.ws._
 import play.api.mvc._
 import utils.ControllerWithBodyParseTolerantJson
 
-import scala.concurrent.{ Await, Future }
+import scala.concurrent.{Await, Future}
 /**
  * Created by jeberle on 25.04.17.
  */
 @Singleton
 class GitController @Inject() (
-    config:                                         play.api.Configuration,
-    jwtVerifier:                                    JWTVerifierProvider,
-    backends:                                       Backends,
-    rmJwtVerifier:                                  ResourcesManagerJWTVerifierProvider,
-    graphMutationClientProvider:                    GraphMutationClientProvider,
-    implicit val wsclient:                          WSClient,
-    protected val orchestrator:                     DatabaseLayer,
-    implicit val graphExecutionContextProvider:     GraphExecutionContextProvider,
-    implicit val janusGraphTraversalSourceProvider: JanusGraphTraversalSourceProvider,
-    implicit val vertexReader:                      VertexReader
-) extends Controller with ControllerWithBodyParseTolerantJson with ControllerWithGraphTraversal with RequestHelper {
-
-  lazy val gc: GraphMutationClient = graphMutationClientProvider.get
+    config:                     play.api.Configuration,
+    jwtVerifier:                JWTVerifierProvider,
+    backends:                   Backends,
+    implicit val wsclient:      WSClient,
+    protected val orchestrator: DatabaseLayer
```

Comment:
suggestion: rename to `databaseAccessLayer`

Reply:
used "dal"